src/main/java/net/dries007/tfc/objects/items/ItemFireClay.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Work under Copyright. Licensed under the EUPL.
- * See the project README.md and LICENSE.txt for more information.
- */
-
-package net.dries007.tfc.objects.items;
-
-import javax.annotation.Nonnull;
-
-import net.minecraft.entity.player.EntityPlayer;
-import net.minecraft.item.ItemStack;
-import net.minecraft.util.ActionResult;
-import net.minecraft.util.EnumActionResult;
-import net.minecraft.util.EnumHand;
-import net.minecraft.world.World;
-
-import net.dries007.tfc.api.capability.size.Size;
-import net.dries007.tfc.api.capability.size.Weight;
-import net.dries007.tfc.client.TFCGuiHandler;
-
-public class ItemFireClay extends ItemTFC
```

Comment:
Why was this moved from a custom item class to an `IPlaceable`? `IPlaceable` is specifically a hack for when we can't / don't want to use a custom item class because of inheritance or inaccessibility reasons - not a general use case for onItemUse.

Reply:
Never mind, I realized its to make it ore dict triggered